astropy/time/tests/test_basic.py

```diff
@@ -600,10 +600,9 @@ def test_init_from_time_objects(self):
 class TestVal2:
     """Tests related to val2"""
 
-    def test_val2_ignored(self):
-        """Test that val2 is ignored for string input"""
-        t = Time('2001:001', 'ignored', scale='utc')
-        assert t.yday == '2001:001:00:00:00.000'
+    def test_unused_val2_ignored(self):
```

Comment:
Out of scope for this PR, and the test docstring no longer matches the actual test (the test does not check for a message that lets the user know we won't use it).

Reply:
Sorry, I was sloppy about reverting this change.